osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceCalculator.cs

```diff
@@ -109,7 +109,9 @@ private double computeAimValue()
             double approachRateBonus = 1.0 + (0.03 + 0.37 * approachRateTotalHitsFactor) * approachRateFactor;
 
             // We want to give more reward for lower AR when it comes to aim and HD. This nerfs high AR and buffs lower AR.
-            if (mods.Any(h => h is OsuModHidden))
+            if (mods.Any(m => m is OsuModBlinds))
+                aimValue *= 1.3 + (totalHits * (0.0016 / (1 + 2 * countMiss)) * Math.Pow(accuracy, 16)) * (1 - 0.003 * Attributes.DrainRate * Attributes.DrainRate);
+            else if (mods.Any(h => h is OsuModHidden))
```

Comment:
Hidden and Blinds are not actually incompatible in game (although maybe they should be?) but the way these bonuses are set up means that they are mutually exclusive. Was this intentional?

Reply:
Yeah, that's right. In the original comment I said I think it's justified since Blinds essentially displaces the visual difficulty of Hidden.